tests/test_rl_strategies.py

```diff
@@ -56,8 +56,8 @@ def power_plant() -> PowerPlant:
 @pytest.mark.parametrize(
     "strategy_class, obs_dim, act_dim, actor_architecture, expected_bid_count, expected_volumes",
     [
-        (RLStrategy, 50, 2, "mlp", 2, [200, 800]),
-        (RLStrategy, 50, 2, "lstm", 2, [200, 800]),
+        (RLStrategy, 38, 2, "mlp", 2, [200, 800]),
+        (RLStrategy, 38, 2, "lstm", 2, [200, 800]),
```

Comment:
why change obs_dim here?

Reply:
Which does not explain why this value is 38 :D
But it is fine if you know and it is not arbitrary